test/FunctionalTests/Microsoft.OData.Core.Tests/Json/JsonWriterAsyncBaseTests.cs

```diff
@@ -321,7 +321,7 @@ public async Task WritesLargeByteArraysCorrectly()
         [Fact]
         public async Task WritesSimpleLargeStringsCorrectly()
         {
-            int inputLength = 1024 * 1024; // 1MB
+            int inputLength = 1024 * 1024;

```

Comment:
Any specific reason you dropped `// 1MB`

Reply:
Not really